web/src/components/ErrorScreen.js

```diff
@@ -1,28 +1,44 @@
 import React from 'react';
 import Guild from './Guild';
 import {UserAvatar} from './DiscordIcon';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
 
 export default class ErrorScreen extends React.Component {
   render() {
     const props = this.makeInner();
     if (!props) return null;
 
-    return (
-      <div className="front flex-vertical flex-spacer">
-        <div className="application-icon">
-          <div className="application-icon-inner" />
-        </div>
-        <div className="front-inner">
-          <header>
-            <h1>{props.header}</h1>
-          </header>
-          <div className="scroller-wrap">
-            <div className="details scroller">{props.body}</div>
+    const screen = (
+      <div key={props.header}>
+        <div className="front flex-vertical flex-spacer">
+          <div className="application-icon">
+            <div className="application-icon-inner" />
+          </div>
+          <div className="front-inner">
+            <header>
+              <h1>{props.header}</h1>
+            </header>
+            <div className="scroller-wrap">
+              <div className="details scroller">{props.body}</div>
+            </div>
+            <footer>{props.footer}</footer>
           </div>
-          <footer>{props.footer}</footer>
         </div>
       </div>
     );
+
+    return (
+      <div>

```

Comment:
so – you're making a `div` here, but `screen` already makes a top level `div`

we can avoid making this extra `div`; see if you can figure out how, otherwise give me a poke

Reply:
Oh yeah this actually ended up being something I tried when I had an issue with the transition not working at all. I'm going to actually try removing it and see if it still works.